### DISCUSSION TITLE
editing a merged branch

### DISCUSSION THREAD

**Discussion by everops-miked** [community/community#38189] [Pull Requests]
I started editing on a branch that got merged before I could save, gave me a 500 when I tried to save as a new branch.

**Comment by ossanna16** [comment]
Hi there @everops-miked and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
